packages/design-system-scripts/cli.js

```diff
@@ -63,18 +63,18 @@ yargs
       yargs.option('rootPath', {
         desc: 'The URL root path for the published docs site.',
         type: 'string',
-        default: '',
+        default: config.rootPath,
       });
     },
     handler: async (argv) => {
       const { buildSrc } = require('./gulp/build');
       const { buildDocs } = require('./gulp/docs');
 
       process.env.NODE_ENV = 'production';
-      if (argv.rootPath !== '') {
+
+      if (argv.rootPath) {
         config.rootPath = argv.rootPath;
       }
-
       if (argv.ignoreRootPath) {
         config.rootPath = '';
       }
```

Comment:
I'd suggest this code and adding this comment 
```
// Allow cli args to override or ignore default rootPath defined in cmsds.config.js
config.rootPath = argv.rootPath
if (argv.ignoreRootPath) {
  config.rootPath = '';
}

Reply:
Thanks, have updated the code.